applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -418,7 +439,9 @@ static void stream_stop_cb(struct bt_audio_stream *stream)
 	int ret;
 
 	LOG_INF("Stream stopped");
-
+#if CONFIG_STREAM_BIDIRECTIONAL
```

Comment:
In the walktrough: need to clarify the use of the CONFIG_STREAM_BIDIRECTIONAL flag here.

Reply:
If CONFIG_STREAM_BIDIRECTIONAL=y, which means the headset could send packet to gateway, then we have to clear the buffer counter once if the stream stopped.

Reply:
If CONFIG_STREAM_BIDIRECTIONAL=y, then headset could send ISO packets to gateway.
However, if disconnection happened, then we have to clean the buffer counter, otherwise the buffer counter will be contains remaining value even if there's no packets in controller buffer.
If CONFIG_STREAM_BIDIRECTIONAL=n, then headset won't send ISO packets, so we don't need to clear the counter.